accessibility_monitoring_platform/settings/base.py

```diff
@@ -240,3 +240,7 @@
 CSP_SCRIPT_SRC = ("'self'",)
 CSP_FONT_SRC = ("'self'",)
 CSP_IMG_SRC = ("'self'", "data:")
+
+AMP_PROTOCOL = os.getenv("AMP_PROTOCOL", "http://")
+AMP_VIEWER_DOMAIN = os.getenv("AMP_VIEWER_DOMAIN", "localhost:8002")
+AMP_PROTOTYPE_NAME = os.getenv("AMP_PROTOTYPE_NAME", "")
```

Comment:
You need to add the environment variables to deploy_feature_to_paas/manifest_template.txt